vaccine_feed_ingest/runners/ny/nyc_arcgis/normalize.py

```diff
@@ -0,0 +1,200 @@
+    appointment_required = site["attributes"]["Intake_ApptRequired"] == "Yes"
+    drop_ins_allowed = site["attributes"]["Intake_WalkIn"] == "Yes"
+
+    return schema.Availability(
+        drop_in=(
+            True if drop_ins_allowed else (False if appointment_required else None)
+        ),
+        appointments=(True if appointment_required else None),
+    )
+
+
+def _get_inventory(site: dict) -> List[schema.Vaccine]:
+    vaccines = []
+
+    if site["attributes"]["ServiceType_JohnsonAndJohnson"] == "Yes":
+        vaccines.append(schema.Vaccine(vaccine="johnson_johnson_janssen"))
+
+    if site["attributes"]["ServiceType_Moderna"] == "Yes":
+        vaccines.append(schema.Vaccine(vaccine="moderna"))
+
+    if site["attributes"]["ServiceType_Pfizer"] == "Yes":
+        vaccines.append(schema.Vaccine(vaccine="pfizer"))
+
+    return vaccines
+
+
+def _get_parent_organization(site: dict) -> Optional[schema.Organization]:
+    # Just the obvious easy cases; Pareto-style
+    name = site["attributes"]["FacilityName"]
+    if name == "Rite Aid Pharmacy":
+        return schema.Organization(name="rite_aid")
+    if name == "Walgreens/Duane Reade":
+        return schema.Organization(name="walgreens")
+    if name == "CVS Pharmacy":
+        return schema.Organization(name="cvs")
+
+    return None
+
+
+def _get_notes(site: dict) -> Optional[List[str]]:
+    note = site["attributes"]["AdditionalInfo"]
+    if note:
+        return [note]
+
+    return None
+
+
+def _get_source(site: dict, timestamp: str) -> schema.Source:
+    source_uri = (
+        "https://services1.arcgis.com/oOUgp466Coyjcu6V/ArcGIS/rest/services"
+        + "/VaccineFinder_Production_View/FeatureServer/0"
+    )
+    return schema.Source(
+        source="nyc_arcgis",
+        id=site["attributes"]["LocationID"],
+        fetched_from_uri=source_uri,
+        fetched_at=timestamp,
+        published_at=datetime.datetime.fromtimestamp(
+            site["attributes"]["LastModifiedDate"] / 1000, utc_tz
+        ).isoformat(),
+        data=site,
+    )
+
+
+def _get_normalized_location(site: dict, timestamp: str) -> schema.NormalizedLocation:
+    return schema.NormalizedLocation(
+        id=f"nyc_arcgis:{site['attributes']['LocationID']}",
+        name=site["attributes"]["FacilityName"],
+        address=schema.Address(
+            street1=site["attributes"]["Address"],
+            street2=site["attributes"]["Address2"],
+            city=site["attributes"]["Borough"],
+            state="NY",
+            zip=site["attributes"]["Zipcode"],
+        ),
+        location=schema.LatLng(
+            latitude=site["geometry"]["y"],
+            longitude=site["geometry"]["x"],
+        ),
+        contact=_get_contacts(site),
+        opening_hours=_get_opening_hours(site),
+        availability=_get_availability(site),
+        inventory=_get_inventory(site),
+        access=schema.Access(wheelchair=site["attributes"]["ADA_Compliant"] == "Yes"),
```

Comment:
Could you set this to one of the values from [the schema](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema#wheelchair-access-enum)? I think you want `yes` or `no` based on this line.

Reply:
i was passing a boolean into this function for the normalized i am working on for GISCorps #144, which it seems like this is doing here. is that incorrect?

Reply:
@MoralCode Yes. See #278 for more info on our upcoming plans for additional validation on this and other fields.